CMSgov/cms-carts-seds
# MDCT-232: Author & Subject Fields for Prince PDF

# Description

Closes [MDCT-232](https://qmacbis.atlassian.net/browse/MDCT-232) by adding Author and Subject fields to the Prince-generated PDF.

## How to test

`cd frontend/react`
`nvm use 14.4.0`
`API_POSTGRES_URL=https://api.mdctcartsdev.cms.gov/ ./env.sh && cp env-config.js public/`
`npm run start` 

* Open your `localhost:3000` in Incognito
* Okta Login
* View CARTS form
* Print -> Entire Form
* View "Document Properties" in Acrobat Reader

Should see "CMS" as Author and "Annual CARTS Report" as Subject :-)

## Dependencies 

See "How to Test" 

# Get it done

PDF is slightly more accessible ~

## Code authors checklist

## Reviewers Checklist (Two different people)

## Assignee
